styles/appc.less

```
@@ -190,14 +194,49 @@
         margin-right: 5px
     }
 
+    .etch-octicon-eye-closed {
+      svg {
+        opacity: 0.5;
+      }
+    }
 
+    .icon-gear {
+      opacity: 0.5;
+    }
     /**
      * android keystore
      */
 
     .keystore-path-input {
         width: 300px;
     }
+
+    &.toolbar-expanded {
+        .icon-gear {
+          opacity: 1;
+
+          &.disabled {
+            color: inherit;
+          }
+        }
+    }
+
+    &.run {
+      .icon-gear {
+        display: none;
+      }
+      .etch-octicon-eye-closed,
+      .etch-octicon-eye {
+        padding: 0 5px;
+      }
+    }
+
+    &.dist-appstore {
```

Would also need the adhoc target added

```suggestion
    &.dist-appstore, &.dist-adhoc {
```